docs/schemaform/form-config.md

```diff
@@ -16,7 +16,8 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
   // The prefix for Google Analytics events that are sent for different form actions
   trackingPrefix: '',
 
-  // The title of the form. Displayed on all pages
+  // The title of the form. Displayed on all pages.
+  // Accepts a string or a function that will be passed `onReview` (with a value of true) as a parameter on the review page.
```

Comment:
Up to you, but I kinda like the format for the page title - i.e., listing both types of title like:
```
 // The title of the chapter
 title: '',
 // This can also be a function that receives an onReview=true parameter only when on the review page
 title: formData => `A title for ${formData.thing}`,
```

Your call 🤷‍♂️

Reply:
On further thought, this is probably an unnecessary / opinionated nit on my part.